projects/1.20/assets/untagged-mobs/untagged_mobs/lang/zh_cn.json

```
@@ -0,0 +1,268 @@
+  "advancements.use_offsetter_flying.title": "心烦意乱",
+  "advancements.use_offsetter.descr": "用偏移器偏移监守者。偏移，偏移，偏移",
+  "advancements.use_offsetter.title": "毫不动摇",
+  "advancements.use_server_key.descr": "使用服务器钥匙打开服务器门",
+  "advancements.use_server_key.title": "还是锁着好",
+  "block.untagged_mobs.alpha_cobblestone": "Alpha圆石",
+  "block.untagged_mobs.alpha_grass_block": "Alpha草方块",
+  "block.untagged_mobs.alpha_missing_block": "§7minecraft:§fmissing_texture_block",
+  "block.untagged_mobs.alpha_server_door": "服务器门",
+  "block.untagged_mobs.alpha_stone": "Alpha石头",
+  "block.untagged_mobs.cardboard_box": "纸箱",
+  "block.untagged_mobs.cheese_block": "奶酪块",
+  "block.untagged_mobs.cheese_bricks": "奶酪砖",
+  "block.untagged_mobs.cheese_door": "奶酪门",
+  "block.untagged_mobs.chiseled_cheese_block": "錾制奶酪块",
+  "block.untagged_mobs.compressing_table": "压缩台",
+  "block.untagged_mobs.dev_block_gray": "灰色开发者方块",
+  "block.untagged_mobs.dev_block_orange": "橙色开发者方块",
+  "block.untagged_mobs.exchanger": "§a交换机",
+  "block.untagged_mobs.fences_missing": "栅栏§7/§f缺失",
+  "block.untagged_mobs.iron_plating": "铁金属板方块",
+  "block.untagged_mobs.mine": "矿井",
+  "block.untagged_mobs.misleading_table": "混沌合成台",
+  "block.untagged_mobs.missing_block": "方块§7/§f缺失",
+  "block.untagged_mobs.missing_block_small": "方块§7.§f小型§7/§f缺失",
+  "block.untagged_mobs.missing_button": "按钮§7/§f缺失",
+  "block.untagged_mobs.missing_fence_gate": "栅栏§8.§r门§7/§f缺失",
+  "block.untagged_mobs.missing_glass": "玻璃§7/§f缺失",
+  "block.untagged_mobs.missing_lamp": "灯§7/§f缺失",
+  "block.untagged_mobs.missing_pressure_plate": "压力§8.§r板§7/§f缺失",
+  "block.untagged_mobs.missing_slab": "台阶§7/§f缺失",
+  "block.untagged_mobs.missing_stairs": "台阶§7/§f缺失",
+  "block.untagged_mobs.missing_wall": "墙§7/§f缺失",
+  "block.untagged_mobs.polished_cheese_block": "磨制奶酪块",
+  "block.untagged_mobs.server_block": "服务器方块",
+  "block.untagged_mobs.server_door": "服务器门",
+  "block.untagged_mobs.vhs_player": "磁带播放器",
+  "block.untagged.compressing_table": "压缩台",
+  "block.untagged.fences_missing": "栅栏§7/§f缺失",
+  "block.untagged.iron_plating": "铁金属板方块",
+  "block.untagged.missing_block": "方块§7/§f缺失",
+  "block.untagged.missing_button": "按钮§7/§f缺失",
+  "block.untagged.missing_fence_gate": "栅栏§8.§r门§7/§f缺失",
+  "block.untagged.missing_lamp": "灯§7/§f缺失",
+  "block.untagged.missing_pressure_plate": "压力§8.§r板§7/§f缺失",
+  "block.untagged.missing_slab": "台阶§7/§f缺失",
+  "block.untagged.missing_stairs": "台阶§7/§f缺失",
+  "block.untagged.missing_wall": "墙§7/§f缺失",
+  "block.untagged.server_block": "服务器方块",
+  "block.untagged.server_door": "服务器门",
+  "block.untagged.vhs_player": "磁带播放器",
+  "death.attack.killing_mace_death": "%1$s被处决了",
+  "death.attack.killing_mace_death.item": "%1$s在尝试逃离%2$s时被%3$s处决",
+  "death.attack.killing_mace_death.player": "%1$s在尝试逃离%2$s时被处决",
+  "death.attack.sweeper_death": "%1$s成为了活靶子",
+  "death.attack.sweeper_death.item": "%2$s使用了%3$s使%1$s成为了活靶子",
+  "death.attack.sweeper_death.player": "%1$s成为了%2$s的活靶子",
```

| **审阅级别** |
|:--------------:|
|⚪ **轻微** / 不阻止合并但需三思|

原文：**%1$s became target practice┃%1$s became target practice for %2$s using %3$s┃%1$s became target practice for %2$s**
```suggestion
  "death.attack.sweeper_death": "%1$s成了活靶子",
  "death.attack.sweeper_death.item": "%2$s使用了%3$s使%1$s成了活靶子",
  "death.attack.sweeper_death.player": "%1$s成了%2$s的活靶子",
```